Python/bytecodes.c

```diff
@@ -2416,6 +2419,9 @@ dummy_func(
             tstate->previous_executor = Py_None;
             Py_INCREF(executor);
             GOTO_TIER_TWO(executor);
+            #else
+            assert(0);
```

Comment:
This should probably be a `Py_FatalError` instead of effectively a `NOP` on debug builds. ;)

Reply:
Interestingly, adding that causes `ENTER_EXECUTOR` to acquire the `HAS_ESCAPES_FLAG` flag. Fixing that (by updating `NON_ESCAPING_FUNCTIONS` in Tools/cases_generator/analyzer.py) causes `_FATAL_ERROR` to lose that flag. I'm going to assume that doesn't matter, because `Py_FatalError` never returns.

Reply:
...now I'm twisting my mind up in knots trying to figure out if `ENTER_EXECUTOR` *should* be considered escaping. But I don't think so (and I don't think it matters very much).